src/richie/apps/courses/models/course.py

```diff
@@ -771,6 +784,19 @@ class CourseRun(TranslatableModel):
         blank=False,
         max_length=20,
     )
+    offer = models.CharField(
+        _("offer"),
+        choices=CourseRunOffer.choices,
```

Comment:
In order to prevent to have to generate a migration each time choices change you could use `lazy`.

```suggestion
        choices=lazy(lambda: CourseRunOffer.choices, tuple)()
```

Reply:
Change done.